### PR TITLE
Fixes the bug of open roll call

### PR DESCRIPTION
**Pull request by BadrTad**
Double bugs:
A null ID gets included in the RPC message, causing the front-end-1 to reject it, the method was incorrect (`message `instead of `broadcast`, it has now been fixed according to the protocol specifications. 
This patch corrects this issue, I'll be providing more tests soon to cover the format specifications 😃

### CODE REVIEW

File: be2-scala/src/main/scala/ch/epfl/pop/json/HighLevelProtocol.scala [hunk 1]
```diff
@@ -163,17 +163,20 @@ object HighLevelProtocol extends DefaultJsonProtocol {
...
-        case Some(idx) => idx.toJson
-        case _ => JsNull
+
+      var jsObjectContent: ListMap[String, JsValue] = ListMap.empty[String, JsValue]
+
+      jsObjectContent += (PARAM_JSON_RPC -> obj.jsonrpc.toJson)
+      jsObjectContent += (PARAM_METHOD -> obj.method.toJson)
+      jsObjectContent += (PARAM_PARAMS -> obj.params.toJson)
+
+      /*Add the id key iif it's non null*/
```
Comment (pierluca) [inline be2-scala/src/main/scala/ch/epfl/pop/json/HighLevelProtocol.scala:173]: `if` or `iff` I assume? :-)

File: be2-scala/src/main/scala/ch/epfl/pop/config/application.conf [hunk 2]
```diff
@@ -1,8 +1,7 @@
 ## Akka configs 
-akka.http.server.idle-timeout =  5 s  
-#akka.http.server.idle-timeout = infinite   # for testing purposes only
-akka.http.client.idle-timeout =  5 s 
 
+akka.http.client.idle-timeout =  600 s 
+akka.http.server.idle-timeout =  600 s
```
Comment (pierluca) [inline be2-scala/src/main/scala/ch/epfl/pop/config/application.conf:4]: is there a reason for switching from 5s to 600s to fix the open roll-call bug?

**Review by pierluca — CHANGES_REQUESTED** [review]
A couple of minor remarks.

Other than that, it seems the commit history is completely mixed with #543 , you might want to clean-up the PR so it does only one thing.

File: be2-scala/src/main/scala/ch/epfl/pop/config/application.conf (hunk 2 above)
Comment (BadrTad) [reply be2-scala/src/main/scala/ch/epfl/pop/config/application.conf:4]: No the timeouts does not make any difference

File: be2-scala/src/main/scala/ch/epfl/pop/json/HighLevelProtocol.scala (hunk 1 above)
Comment (BadrTad) [reply be2-scala/src/main/scala/ch/epfl/pop/json/HighLevelProtocol.scala]: yes it should be iff

**Review by BadrTad — COMMENTED** [review]
The github history has been cleaned.

**Review by pierluca — APPROVED** [review]
Very clean ! 👍 
LGTM

**Pull request merged**